cpp/tests/bitmask/nans_to_nulls_tests.cu

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cudf/copying.hpp>
+#include <cudf/table.hpp>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <tests/utilities/column_wrapper.cuh>
+#include "tests/utilities/compare_column_wrappers.cuh"
+#include <tests/utilities/cudf_test_fixtures.h>
+#include <tests/utilities/cudf_test_utils.cuh>
+#include <tests/utilities/compare_column_wrappers.cuh>
+#include <cudf/types.hpp>
+#include <utilities/wrapper_types.hpp>
+
+#include <random>
+#include <cudf/functions.h>
+
+using bit_mask::bit_mask_t;
+
+template <typename T>
+struct BitMaskTest : GdfTest {};
+
+using test_types =
+    ::testing::Types<float, double>;
+TYPED_TEST_CASE(BitMaskTest, test_types);
+
```

Comment:
Need a test of empty input column, as well as error conditions (unsupported type).